myapp/app/models/task_label.rb

```diff
@@ -1,4 +1,7 @@
 class TaskLabel < ApplicationRecord
+  validates :task_id, presence: true
+  validates :label_id, presence: true
```

Comment:
you don't need these as you already has the `belong_to` association

Reply:
Thanks for this information, I will remove this two lines